docker-compose.yml

```diff
@@ -112,7 +112,7 @@ services:
   #     LOGGING_QUIET: 'true'
 
   matomodb:
-    image: mariadb:11.4
+    image: mariadb:10.6
```

Comment:
Make sure this file only contains changes that you are making. For example, the Matomo updates need to remain the same

Reply:
Not sure why they got reverted. I'll revert that change.

Reply:
Changes fixed in 517ffaf9fcf5e76929e9409a8e8dca027af75ac5